permissions/plugin-pipeline-milestone-step.yml

```diff
@@ -4,4 +4,9 @@ paths:
 - "org/jenkins-ci/plugins/pipeline-milestone-step"
 developers:
 - "amuniz"
+- "svanoort"
+- "abayer"
 - "jglick"
+- "rsandell"
+- "stephenc"
```

Comment:
@batmat  Oopsy, that's a gotcha!  Lesson learned about trying to do more than the minimum, especially when trying to ship fixes on a timeline.